lib/devise-security/models/secure_validatable.rb

```diff
@@ -99,8 +113,30 @@ def email_required?
         true
       end
 
+      def allow_passwords_equal_to_email
+        self.class.allow_passwords_equal_to_email
+      end
+
+      def email_validation
+        self.class.email_validation
+      end
+
+      def password_complexity
+        self.class.password_complexity
+      end
+
+      def password_length
+        self.class.password_length
+      end
```

Comment:
It may be possible to just delegate these calls to the class instead of defining new methods.

Reply:
Updated to use this!